src/cascadia/TerminalSettingsEditor/SettingContainer.cpp

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "pch.h"
+#include "SettingContainer.h"
+#include "SettingContainer.g.cpp"
+#include "LibraryResources.h"
+
+using namespace winrt::Windows::UI::Xaml;
+
+namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
+{
+    DependencyProperty SettingContainer::_HeaderTextProperty =
+        DependencyProperty::Register(
+            L"HeaderText",
+            xaml_typename<hstring>(),
+            xaml_typename<Editor::SettingContainer>(),
+            PropertyMetadata{ nullptr });
+
+    DependencyProperty SettingContainer::_HelpTextProperty =
+        DependencyProperty::Register(
+            L"HelpText",
+            xaml_typename<hstring>(),
+            xaml_typename<Editor::SettingContainer>(),
+            PropertyMetadata{ box_value(L"") });

```

Comment:
technically these are static initializers, and I am loathe to allow them. Can we find a way to not initialize them on library load?